1_create_package/nomad/conf.d/server.conf

```diff
@@ -3,8 +3,6 @@ bind_addr = "0.0.0.0" # the default
 
 advertise {
   # Defaults to the first private IP address.
-  http = "192.168.2.113"
```

Comment:
I don't have this file at all in my repo. Is that something new?

Reply:
It is, i've diverged pretty heavily, some of this was cruft from testing